content/vault/v1.21.x (rc)/content/docs/sysadmin/snapshots/recover-a-secret/single-cluster.mdx

```diff
@@ -31,6 +31,37 @@ Load the snapshot holding the secret you want to recover.
 
 <Tabs>
 
+<Tab heading="Web UI" group="ui">
```

Comment:
It looks like all the steps on this pages are the same as the replicated cluster instructions? If so, we should move them into partials (like the CLI and API directions) so we can use them across both pages